src/taxonomy/taxonomy-detail/TaxonomyDetailPage.jsx

```diff
@@ -104,8 +105,15 @@ const TaxonomyDetailPage = () => {
     );
   };
 
+  const getSystemDefinedBadge = () => {
+    if (taxonomy.systemDefined) {
+      return <SystemDefinedBadge taxonomyId={taxonomyId} />;
+    }
+    return undefined;
+  };
+
   return (
-    <>
+    <div className="taxonomy-detail-page">
```

Comment:
I think this class isn't in use anymore, so this change can be reverted.

Reply:
Updated https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/dab9d8a01a0389826e989252b112c3b70869e251